src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/ios/LoginPage.java

```diff
@@ -15,6 +15,8 @@
 @DeviceType(pageType = Type.IOS_PHONE, parentClass = LoginPageBase.class)
 public class LoginPage extends LoginPageBase {
 
+	private static final String THIS_METHOD_IS_DEFINED_ONLY_IN_ANDROID = "This method is not yet implemented for iOS";
```

Comment:
A better way I think will be next:
1. create a new class - ProjectConstant
2. define in this class all constants like
public static final String UNIMPLEMENTED_IOS = "Unimplemented for iOS";
public static final String UNIMPLEMENTED_ANDROID = "Unimplemented for Android";
public static final String UNIMPLEMENTED_DESKTOP = "Unimplemented for Desktop";

3. use these constants in the project
4. in the future define another constants for project here